menus.js

```diff
@@ -178,31 +178,88 @@ function macMenuHelp ({i18n, shell}) {
   };
 }
 
-menuTemplates.mac = ({dialog, i18n, app, checkNewUpdates, extraMenus, mainWindow, config, shell}) => [
+menuTemplates.mac = ({dialog, i18n, app, checkNewUpdates, extraMenus, mainWindow, config, shell, isInspector}) => [
```

Comment:
rather than `isInspector`, a variable name that describes what will actually happen when it is sent would be better, like `shouldShowFileMenu`. that way this project doesn't have to know about inspector vs not inspector.